packages/@aws-cdk/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -60,6 +60,17 @@ export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationL
    */
   readonly memoryLimitMiB?: number;
 
+  /**
+   * The amount (in GiB) of ephemeral storage to be allocated to the task. The maximum supported value is 200 GiB.
+   *
+   * NOTE: This parameter is only supported for tasks hosted on AWS Fargate using platform version 1.4.0 or later.
+   *
+   * This default is set in the underlying FargateTaskDefinition construct.
+   *
+   * @default 20
+   */
+  readonly ephemeralStorageGiB?: number;
```

Comment:
I think it would be more consistent if you would use the Size class of cdk.

Reply:
That would be inconsistent with [FargateTaskDefinition#ephemeralStorageGiB](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-ecs.FargateTaskDefinition.html#ephemeralstoragegib), which this code depends on, that also uses `number`.

The only unit supported by ECS/Fargate is GiB, and the only valid values are 21 GiB to 200 GiB (in increments of 1 GiB), so I'm not sure supporting other units would be useful.